cms/static/sass/_base.scss

```diff
@@ -666,67 +666,63 @@ hr.divider {
   }
 }
 
-// +JS Dependent
-// ==================== 
-body.js {
```

Comment:
Does removing this affect anything across the platform that may use the `.js` class?